src/sharedData/visualization/components/VisualizationWrapper.js

```diff
@@ -98,14 +100,17 @@ const VisualizationWrapper = props => {
     imagePrinter.printMap('CurrentSize', mapTitle);
   }, [imagePrinter, mapTitle]);
 
-  const onDelete = () => {
+  const onDelete = useCallback(() => {
     dispatch(deleteVisualizationConfig(data)).then(data => {
       const success = _.get('meta.requestStatus', data) === 'fulfilled';
       if (success) {
         onDeleted();
+        trackEvent('dataMap.delete', {
+          props: { owner: owner.slug, title: mapTitle },
```

Comment:
question: do we need the map title? won't that identify the user? @DerekCaelin

Reply:
I don't think that is user related data, it is the data map title, I think we are tracking the names of groups too in plausible

Reply:
Groups have many users. A map is created by a single user. (Per Derek below, okay to omit.)